src/homework/db/crud.py

```diff
@@ -78,3 +78,23 @@ def delete_application(app_id: str) -> bool:
     except (exc.SQLAlchemyError, exc.DBAPIError):
         return False
     return True
+
+
+def get_user_balance_by_id(user_id: int) -> None | int:
+    """
+    Get the user's balance from the database by their id.
+
+    If the user with given user ID does not exist, function returns None.
```

Comment:
maybe change user ID to user_id for consistency?

Reply:
Yes, it is a good point.